stdlib/public/SwiftOnoneSupport/SwiftOnoneSupport.swift

```diff
@@ -273,17 +273,15 @@ extension Range where Bound: Strideable, Bound.Stride : SignedInteger {
   func _prespecializeIntegerRange(bound: Bound) {}
 }
 
-extension ClosedRange {
+extension ClosedRange where Bound: Strideable, Bound.Stride : SignedInteger {
   // init(uncheckedBounds: (lower: A, upper: A)) -> Swift.ClosedRange<A>
   @_semantics("prespecialize.$sSN15uncheckedBoundsSNyxGx5lower_x5uppert_tcfC")
```

Comment:
```
$sSN15uncheckedBoundsSNyxGx5lower_x5uppert_tcfC ---> Swift.ClosedRange.init(uncheckedBounds: (lower: A, upper: A)) -> Swift.ClosedRange<A>
```

I think this one needs to be put back into an unconditional extension with a new prespecialization point.

Reply:
Oh, indeed, well noticed!